Files used for paper: Microbiome-driven machine learning for predicting suppressiveness to Rhizoctonia solani in organic-amended soils

[Dataset - 1](https://github.com/vbuttros/rsolani_severity/tree/88d63dd2f2ae5e0596214ed948b07f6aa549044e)

[Dataset - 2](https://github.com/vbuttros/rsolani_severity/tree/e18fd676335502ca12d94c98e53a4bd7089b19ec)

Scripts for the pre-processing stage of each dataset


[For dataset 1](https://github.com/vbuttros/rsolani_severity/blob/main/df1_pre_process.ipynb)

[For dataset 2](https://github.com/vbuttros/rsolani_severity/blob/main/df2_pre_process.ipynb)

Merger and ML implementation. Uses the outputs from the pre-process stage.

[Merger](https://github.com/vbuttros/rsolani_severity/blob/main/merger_predictor_rsolani%20.ipynb)

UPCOMING FEATURE:
Trained random forest model with 16S and ITS sequences and the associated feature importance for Rhizoctonia solani suppression

A database of sequencing reads and their impact on disease suppression, allowing other researchers to identify microorganisms associated with Rhizoctonia solani suppression in their own samples. This resource will link 16S and ITS reads to their potential role in disease suppression, helping to uncover key microbial taxa that contribute to soil health and pathogen control.
